projects/ngx-location-viewer/src/lib/components/ngx-location-viewer.component.html

```diff
@@ -1,19 +1,25 @@
-<aui-leaflet [leafletMap]="leafletMap" [hasSidebar]="hasSidebar">
-  <div *ngIf="hasSidebar">
-    <ng-content></ng-content>
-  </div>
-  <div *ngIf="showLayerManagement && hasSidebar">
-    <aui-layer-management [supportingLayers]="supportingLayers" [operationalLayer]="operationalLayer"
-      (layerVisibilityChange)="handleLayerVisibilityChange($event)" (layerManagementClosed)="toggleLayermanagement()">
-    </aui-layer-management>
+<aui-leaflet [leafletMap]="leafletMap" [hasSidebar]="showSidebar">
+  <div class="sidebar" *ngIf="hasSidebar && showSidebar">
+    <div class="sidebar__header">
+      <button class="a-button-transparent a-button--default has-icon sidebar__close-button" aria-label="Sluiten"
+        (click)="toggleSidebar()">
+        <aui-icon name="ai-close"></aui-icon>
+      </button>
+    </div>
+    <div class="sidebar__content">
+      <ng-content></ng-content>
+      <aui-layer-management *ngIf="showLayerManagement" [supportingLayers]="supportingLayers" [operationalLayer]="operationalLayer"
+        (layerVisibilityChange)="handleLayerVisibilityChange($event)">
+      </aui-layer-management>
+    </div>
   </div>
   <div class="controls-top-left" controls top left>
-    <button *ngIf="showLayerManagement" type="button"
+    <button *ngIf="hasSidebar" type="button"
       class="a-button a-button--small has-icon a-button--block u-margin-bottom-xs" name="toggle-layermanagement"
-      [title]="hasSidebar ? 'Sluit lagenoverzicht' : 'Open lagenoverzicht'"
-      [attr.aria-label]="hasSidebar ? 'Sluit lagenoverzicht' : 'Open lagenoverzicht'" (click)="toggleLayermanagement()">
-      <aui-icon *ngIf="hasSidebar" name="ai-arrow-button-left-1"></aui-icon>
-      <aui-icon *ngIf="!hasSidebar" name="ai-arrow-button-right-1"></aui-icon>
+      [title]="showSidebar ? 'Sluit zijbalk' : 'Open zijbalk'"
+      [attr.aria-label]="showSidebar ? 'Sluit zijbalk' : 'Open zijbalk'" (click)="toggleSidebar()">
```

Comment:
Thanks for not forgetting the ARIA label ;-) Would it be much effort to also refactor these labels, so that they can be overwritten? Now it is hard-coded Dutch, but there are use cases where it could be another language.

The ideal way would be to pass one object to the component in which the developer can overwrite the default Dutch translations. An example can be found here: https://github.com/digipolisantwerp/antwerp-ui_angular/blob/master/packages/ngx-user-menu/src/lib/components/user-menu/user-menu.component.ts (line 22)